<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>The game of life</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ico.bmp</normaloff>ico.bmp</iconset>
  </property>
  <property name="windowOpacity">
   <double>29.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>651</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignTop">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Step:</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QLabel" name="activeLabelStep">
        <property name="text">
         <string notr="true">0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignTop">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSlider" name="speedSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>true</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>x1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>x2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>x3</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>x4</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionSave_XML_file"/>
    <addaction name="actionLoad_XML_file"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRandomize"/>
    <addaction name="separator"/>
    <addaction name="actionChange_amount_to_revive"/>
    <addaction name="actionChange_amount_to_survive"/>
    <addaction name="actionChange_amount_of_points"/>
    <addaction name="separator"/>
    <addaction name="actionChange_alive_cell_color"/>
    <addaction name="actionChange_dead_cell_color"/>
    <addaction name="actionChange_grid_color"/>
    <addaction name="separator"/>
    <addaction name="actionChange_size"/>
    <addaction name="actionChange_radius"/>
    <addaction name="separator"/>
    <addaction name="actionChange_language"/>
   </widget>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>Information</string>
    </property>
    <addaction name="actionTutorial"/>
    <addaction name="actionAuthor"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuSettings"/>
   <addaction name="menuInformation"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F3</string>
   </property>
  </action>
  <action name="actionRandomize">
   <property name="text">
    <string>Randomize</string>
   </property>
  </action>
  <action name="actionChange_amount_to_revive">
   <property name="text">
    <string>Change amount to revive</string>
   </property>
   <property name="toolTip">
    <string>Amount of neighbours to live.</string>
   </property>
  </action>
  <action name="actionChange_amount_to_survive">
   <property name="text">
    <string>Change amount to survive</string>
   </property>
   <property name="toolTip">
    <string>Amount of neighbours to die.</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actionChange_amount_of_points">
   <property name="text">
    <string>Change number of points</string>
   </property>
  </action>
  <action name="actionChange_alive_cell_color">
   <property name="text">
    <string>Change alive cell color</string>
   </property>
  </action>
  <action name="actionChange_dead_cell_color">
   <property name="text">
    <string>Change dead cell color</string>
   </property>
  </action>
  <action name="actionChange_size">
   <property name="text">
    <string>Change size</string>
   </property>
  </action>
  <action name="actionChange_grid_color">
   <property name="text">
    <string>Change grid color</string>
   </property>
  </action>
  <action name="actionSave_XML_file">
   <property name="text">
    <string>Save XML file</string>
   </property>
  </action>
  <action name="actionLoad_XML_file">
   <property name="text">
    <string>Load XML file</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionChange_language">
   <property name="text">
    <string>Change language</string>
   </property>
  </action>
  <action name="actionChange_radius">
   <property name="text">
    <string>Change radius</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
